databases/gorm/categories.go

```diff
@@ -1,25 +1,61 @@
 package database
 
-import "github.com/arshamalh/blogo/models"
+import (
+	"errors"
+	"log"
+
+	"github.com/arshamalh/blogo/models"
+	"gorm.io/gorm"
+)
 
 func (gdb *gormdb) CheckCategoryExists(name string) bool {
-	results := gdb.db.Take(&models.Category{}, "name = ?", name)
-	return results.RowsAffected > 0
+	var category models.Category
+	err := gdb.db.Where("name = ?", name).First(&category).Error
+	exists := !errors.Is(err, gorm.ErrRecordNotFound)
+
+	if exists {
+		log.Printf("Category with name '%s' exists.", name)
+	} else {
+		log.Printf("Category with name '%s' does not exist.", name)
+	}
+
+	return exists
 }
 
 func (gdb *gormdb) CreateCategory(catg *models.Category) (uint, error) {
 	err := gdb.db.Create(&catg).Error
+
+	if err == nil {
+		log.Printf("Created category with ID: %d", catg.ID)
+	} else {
+		log.Printf("Failed to create category. Error: %v", err)
+	}
+
```

Comment:
Writing extra conditions just for logging is extra work.

Reply:
ok